agenta-backend/agenta_backend/models/api/evaluation_model.py

```diff
@@ -71,13 +71,18 @@ class EvaluationScenario(BaseModel):
     id: Optional[str]
 
 
+class AICritiqueCreate(BaseModel):
+    outputs: List[EvaluationScenarioOutput]
+    evaluation_prompt_template: Optional[str]
+    open_ai_key: Optional[str]
+
+
 class EvaluationScenarioUpdate(BaseModel):
     vote: Optional[str]
     score: Optional[str]
+    ai_critique_score: Optional[str]
```

Comment:
why is this different than score?

Reply:
No difference. I have effected it to be score.